### PR TITLE
Move to current session tab when switching schedules screen

### LINKED ISSUE #589: When switching time based sessions, I want to see current time session
## Overview (Required)
- Title says everything.
- You can check it by changing device time
image

### PR DESCRIPTION
**Pull request by seisuke**
## Issue
- close #589 

## Overview (Required)
- When switching time based sessions, I want to see current time session

## Links
-

## Screenshot
Before | After
:--: | :--:
before | after

### CODE REVIEW

**Comment by s-aska** [comment]
👍

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt [hunk 1]
```diff
@@ -120,7 +120,9 @@ class SessionsFragment : DaggerFragment(), Findable, OnReselectedListener {
         sessionsViewModel.tab.observe(this, { result ->
             when (result) {
                 is Result.Success -> {
-                    sessionsViewPagerAdapter.setSessionTab(result.data)
+                    sessionsViewPagerAdapter.setSessionTab(result.data) {
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt]: How about this?

```
if(result.data is SessionTab.Schedule) scrollToRecentScheduleTab()
```

**Comment by takahirom** [comment]
Thanks!

**Comment by takahirom** [comment]
LGTM!
Thank you this is useful feature!

**Comment by takahirom** [comment]
This implementation will scroll to the tab of the current time when registering in favorites, so we will revert

**Comment by seisuke** [comment]
I will fix it

**Pull request merged**